### ISSUE TITLE
Code

### ISSUE THREAD

**Issue by marvellee1** [NiccoloCavagnero/EntropicScore#1]
Will the code be open source?

**Comment by NiccoloCavagnero** [comment]
Thank you for your interest. I am not planning to release the code for now, however the metric computation is straightforward:

1) replace norm layers such that their output is in [0,1], by dividing for the channel-wise maximum;
2) for each normalised activation, compute - torch.mean(activation * torch.log(activation + epsilon)).
3) forward the input and sum the scores across the different activations.
